### PR TITLE
Google buffer

### PR DESCRIPTION
**Pull request by armstrmi**
## Description of Changes
* Included functionality for flushing buffer upon close
* Added tests for testing the closing of the google-cloud-buffer
## **Please check that the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Add a changelog entry (for non-trivial bug fixes / features)
- [ ] CI passes

### CODE REVIEW

**Review by cpheps — APPROVED** [review]
LGTM

File: operator/builtin/output/forward/forward.go [hunk 1]
```diff
@@ -106,7 +106,7 @@ func (f *ForwardOutput) Stop() error {
 
 	entries, err := f.buffer.Close()
 	if err != nil {
-		f.Errorf("Failed to retreive entries")
+		f.Error("Failed to retreive entries")
```
Comment (BinaryFissionGames) [inline operator/builtin/output/forward/forward.go:109]:
```suggestion
		f.Error("Failed to retrieve entries")
```

File: operator/builtin/output/googlecloud/operator.go [hunk 2]
```diff
@@ -73,12 +73,23 @@ func (g *GoogleCloudOutput) Stop() error {
...
-	if _, err := g.buffer.Close(); err != nil {
+	entries, err := g.buffer.Close()
+	if err != nil {
 		return fmt.Errorf("failed to close buffer: %w", err)
 	}
 	g.Debug("Closed buffer")
 
+	//checks if buffer was empty, if not send requests
+	if len(entries) != 0 {
+		ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
```
Comment (BinaryFissionGames) [inline operator/builtin/output/googlecloud/operator.go:84]: Small question here, why 10 seconds here, but 5 seconds in other operators?

**Review by BinaryFissionGames — COMMENTED** [review]
2 really small things

**Review by BinaryFissionGames — APPROVED** [review]

**Pull request merged**